analysis_options.yaml

```diff
@@ -22,8 +22,8 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
-    # avoid_print: false  # Uncomment to disable the `avoid_print` rule
-    # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule
-
+    avoid_print: false  # Uncomment to disable the `avoid_print` rule
```

Comment:
Print statements should not be present in the production code. Remove this lint check. If you want to use print then use the debugPrint method for logs.